cEP-0025.md

```diff
@@ -0,0 +1,169 @@
+# Integration of pyflakes-enhanced-AST into coala
+
+| Metadata |                                                 |
+| -------- | ----------------------------------------------- |
+| cEP      | 22                                              |
+| Version  | 1.0                                             |
+| Title    | Integration of pyflakes-enhanced-AST into coala |
+| Authors  | Ankit Joshi <mailto:ajankit2304@gmail.com>      |
+| Status   | Proposed                                        |
+| Type     | Feature                                         |
+
+## Abstract
+
+This document describes how to create a metabear to integrate
+pyflakes-enhanced-AST into coala and demonstrates its use by
+creating a simple dependent bear.
+
+## Introduction
+
+Flake8 is the most commonly used linter framework when it comes to static
+code analysis for python. It basically wraps around the tools pyflakes,
+pycodestyle and mccabe scripts because each of those projects
+tend to add new checks in minor releases. That way if you install something
+from the 2.5 series today, you will not find new checks in the same series
+in a month from now when you install it again.
+
+It even provides with a plugin mechanism. Additional linters are
+supported using the flake8 plugin system, and may be Python AST
+based or built using pycodestyle's internal logical-line-based
+checker API (checkout [Hacking](https://github.com/openstack-dev/hacking)).
+
+There has not been an option to use the pyflakes internal AST based checker
+API to create custom linters. Thus, the full potential of enhanced AST isn't
```

Comment:
`enhanced AST` here vs `pyflakes internal AST` on the previous line.

The reader may not get the leap from `pyflakes internal AST` to `enhanced AST` being the same thing.